config/routes.rb

```diff
@@ -1,4 +1,22 @@
 Rails.application.routes.draw do
+  # get 'entities/index'
+  # get 'group/index'
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.

Reply:
This issue is well resolved! ✔️